### PR TITLE
define Signal natives in Java;  implement findSignal0

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

File: java.base/src/main/java/jdk/internal/misc/Signal$_native.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                case "QUIT": return 3;
+                case "ILL": return 4;
+                case "ABRT": return 6;
+                case "FPE": return 8;
+                case "KILL": return 9;
+                case "SEGV": return 11;
+                case "PIPE": return 13;
+                case "ALRM": return 14;
+                case "TERM": return 15;
+                default: return -1;
```
Comment (dmlloyd) [inline java.base/src/main/java/jdk/internal/misc/Signal$_native.java]: These signal numbers *should* come from constants in `org.qbicc.runtime.posix.Signal` and `org.qbicc.runtime.stdc.Signal`.

File: java.base/src/main/java/jdk/internal/misc/Signal$_native.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            }
+        } else if (Build.Target.isWindows()) {
+            switch(sigName) {
+                case "ABRT": return 22;
+                case "FPE": return 8;
+                case "SEGV": return 11;
+                case "INT": return 2;
+                case "TERM": return 15;
+                case "BREAK": return 21;
+                case "ILL": return 4;
```
Comment (dmlloyd) [inline java.base/src/main/java/jdk/internal/misc/Signal$_native.java]: Most of these are defined in `org.qbicc.runtime.stdc.Signal` but I'm not sure where to find the "signal number" for `BREAK` on Windows.

File: java.base/src/main/java/jdk/internal/misc/Signal$_native.java (hunk 1 above)
Comment (dgrove-oss) [reply java.base/src/main/java/jdk/internal/misc/Signal$_native.java]: Nice!  I didn't think to look for real constants....fixing

Comment (dgrove-oss) [reply java.base/src/main/java/jdk/internal/misc/Signal$_native.java]: Fixed & squashed.

File: java.base/src/main/java/jdk/internal/misc/Signal$_native.java (hunk 2 above)
Comment (dgrove-oss) [reply java.base/src/main/java/jdk/internal/misc/Signal$_native.java]: Left BREAK as a magic number; used proper constants for the rest.

**Review by dmlloyd — APPROVED** [review]

**Pull request merged**